stl/inc/algorithm

```diff
@@ -1865,6 +1865,199 @@ _NODISCARD _FwdIt1 find_end(_ExPo&& _Exec, _FwdIt1 _First1, _FwdIt1 _Last1, _Fwd
 }
 #endif // _HAS_CXX17
 
+#ifdef __cpp_lib_concepts
+namespace ranges {
+    // CLASS ranges::_Find_end_fn

```

Comment:
this feels like a pointless comment?

Reply:
Ouch. Yes, I've been putting only the `// VARIABLE ranges::find_end` banner for the public entity on the rest of these algorithm niebloids. I'll move that comment up here.